lib/ImageProcessor.js

```diff
@@ -85,13 +88,14 @@ class ImageProcessor {
                 imageData.fileName.indexOf(resize.directory) !== 0; // don't process images in the output folder
         }).forEach((resize) => {
             resize.acl = resize.acl || acl;
+            resize.cacheControl = (resize.cacheControl !== undefined) ? resize.cacheControl : cacheControl;
             resize.bucket = resize.bucket || bucket;
             resize.jpegOptimizer = resize.jpegOptimizer || jpegOptimizer;
             resize.optimizerOptions = optimizerOptions;
 
             promise = promise
               .then(() => this.execResizeImage(resize, imageData))
-              .then((image) => this.fileSystem.putObject(image))
+              .then((image) => this.fileSystem.putObject(image, resize))
```

Comment:
`resize` will have much information. I suggest `{cacheControl: resize.cacheControl}` is enough to pass and makes clearly.